client/my-sites/people/people-list-section-header/index.jsx

```diff
@@ -15,37 +17,36 @@ import Gridicon from 'components/gridicon';
 import Tooltip from 'components/tooltip';
 import config from 'config';
 
-export default React.createClass( {
-	displayName: 'PeopleListSectionHeader',
-
-	PropTypes: {
+class PeopleSectionHeader extends Component {
+	static propTypes = {
+		translate: PropTypes.func,
 		label: React.PropTypes.string.isRequired,
 		count: React.PropTypes.number,
 		isFollower: React.PropTypes.bool,
-		site: React.PropTypes.object
-	},
+		site: React.PropTypes.object,
+	};
 
-	getInitialState() {
-		return {
-			addPeopleTooltip: false
-		}
-	},
+	static defaultProps = {
+		translate: identity,
+		isFollower: false,
+	};
 
-	getDefaultProps() {
-		return {
-			isFollower: false
+	constructor() {
+		super( ...arguments );
```

Comment:
I find the use of `arguments` here curious. `arguments` is characteristically different than normal props. Is there a specific reason to break from the React convention which passes in a single object argument `props`?

```js
constructor( props ) {
	super( props );
}
```

Also, I have recently seen some people skip the `constructor()` when only needing to assign initial state (independent of props) and instead using static class properties…

```js
class Foo {
	state = {
		addPeopleTooltip: false,
	};
}
```